rebar.config

```diff
@@ -1,8 +1,8 @@
 %% -*- mode: erlang;erlang-indent-level: 4;indent-tabs-mode: nil -*-
-{require_otp_vsn, "R16|17|18|19"}.
+{require_otp_vsn, "R16|17|18|19|20|21|22"}.
 {erl_opts,
  [debug_info,
-  {platform_define, "^19.", 'ERLANG_OTP_VERSION_19'},
+  {platform_define, "^(19|2[0-9])", 'ERLANG_OTP_VERSION_19'},
```

Comment:
Seems like this makes the macro name a little misleading. Maybe `ERLANG_19_PLUS` would be more succinct and accurate?